.github/workflows/CI-tests.yml

```diff
@@ -41,13 +50,25 @@ jobs:
         run: |
           poetry config virtualenvs.in-project true
           poetry install
+          poetry add django==${{ matrix.django-version }}
+          poetry show django
```

Comment:
is this a remnant of your debugging? is it still needed?

Reply:
The `poetry add django` line is still needed, as otherwise we won't have the right version of Django in the container, given that the pyproject.toml file specifies a version range.

The `poetry show django` line is to prove that it has worked. We could conceivably remove that line, but then, if the tree falls in the forest, and nobody reads it, does it still pass?

Reply:
The same is true of the `poetry run django --version` line further down.

Reply:
<silent tree fall>

👍